hoot-core/src/main/cpp/hoot/core/ops/CalculateStatsOp.cpp

```diff
@@ -357,12 +357,15 @@ void CalculateStatsOp::apply(const OsmMapPtr& map)
     _addStat("Percentage of Total Features Unmatched",
              ((double)unconflatedFeatureCount / (double)featureCount) * 100.0);
 
+    QStringList featureTypesToSkip;
```

Comment:
Small change that made it easier to debug some map consumption bugs I had introduced.